core/services/workflows/models.go

```diff
@@ -0,0 +1,205 @@
+package workflows
+
+import (
+	"fmt"
+
+	"github.com/dominikbraun/graph"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type stepRequest struct {
+	stepRef string
+	state   executionState
+}
+
+// stepDefinition is the parsed representation of a step in a workflow.
+//
+// Within the workflow spec, they are called "Capability Properties".
+type stepDefinition struct {
```

Comment:
Is models.go moved from workflow.go? What is confusing to me that when I look at the commits separately, none of them deletes workflow.go ... :|  It's a bit hard for me to understand what changed. Can you maybe split this PR into pure file rename + actual changes separately?

Reply:
Yeah, sorry about that. I made the rename be the last commit. The meaty portion of the PR is in the first commit, the refactor into two files is the second commit.